lib/component.js

```diff
@@ -15,6 +15,12 @@ const DOCUMENT_FRAGMENT_NODE = 11;
  *
  * @example <caption>Defining a Panel component</caption>
  * class MyWidget extends Component {
+ *   static get attrsSchema() {
```

Comment:
I get the desire to document usage, but considering that this example is so bare-bones and doesn't even have `config` details, I'd prefer to leave it out of this example and document at the implementation point.